components/observatorium-configure.libsonnet

```diff
@@ -35,8 +35,8 @@
     config+:: obs.config.query,
   },
 
-  queryCache+:: {
-    config+:: obs.config.queryCache,
+  queryFrontend+:: {
```

Comment:
Probably operator needs an update https://github.com/observatorium/deployments/blob/1964b4e5e797c8e77a86bc17c61ba855c35dfad8/operator/jsonnet/operator-config.libsonnet#L55

Reply:
Umm, do we want to use the old name `queryCache`?

Reply:
Thanks, I missed that. Let me change it.